.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: "Release"
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  run-tests:
+    name: "Run tests"
+    uses: ./.github/workflows/tests.yml
+    secrets: inherit
+
+  release:
+    needs: run-tests
+    name: "Release"
+    runs-on: ubuntu-20.04
+    permissions: write-all
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: npm install @semantic-release/exec@6.0.3 -D
```

Comment:
Try to install using action in the next task

Reply:
@bissquit , is [this](https://github.com/cloud-labs-infra/helm-certificate-issuers/pull/1/commits/5dce8df13b61857d2bf492cd5ddc8935576c545d) smth you are looking for?